docs/src/main/java/GettingStarted.java

```diff
@@ -97,13 +100,66 @@ public void showTsaApi() throws Exception {
     }
   }
 
+  @Test
+  public void showDynamicTsaApi() throws Exception {
+    // tag::showDynamicTsaApi[]
+    ConfigurationContext configContext = customConfigurationContext()
+        .tsa(tsa -> tsa
+            .topology(
+                new Topology(
+                    distribution(version("3.9-SNAPSHOT"), KIT, TERRACOTTA_OS),
```

Comment:
there's an inconsistency regarding the version, above it is 3.8.1 and here 3.9-SNAPSHOT

Reply:
Dynamic config changes are available in certain kits only (e.g. ehcache 3.9-SNAPSHOT, tc-ee 10.7.0-SNAPSHOT). Using these kits automatically translates to `Distribution107Controller`. Using a global `EHCACHE_OS_VERSION` didn't seem correct, because the added tests need to use a different ehcache version